### PR TITLE
#896: Test Swin transformer TT sub-modules: Encoder, Embedding, Patch…

### PR DESCRIPTION
**Pull request by Sudharsan-V**
This branch contains the following:
TT implementation of the Swin transformer's sub-modules namely Encoder, Embeddings, PatchEmbeddings, PatchMerging, and Stage. This branch also has the test files for all the sub-modules mentioned.

### CODE REVIEW

**Review by ashayestehmanesh — CHANGES_REQUESTED** [review]

**Review by farbabi — APPROVED** [review]
@Sudharsan-V approved assuming cpu and tt demos come next.

**Review by ashayestehmanesh — APPROVED** [review]
Please address my comment before merging. Make sure that part of the code is not invoked. 

adding suggested assert and re-running your tests is enough assurance. Thank you :)

**Pull request merged**
